docs/spec/manifest-v2-2.md

```diff
@@ -89,6 +89,17 @@ image manifest based on the Content-Type returned in the HTTP response.
             The os field specifies the operating system, for example
             `linux` or `windows`.
 
+        - **`os.version`** *string*
+
+            The optional os.version field specifies the operating system version,
+            for example `10.0.10586`.
+
+        - **`os.features`** *array*
+
+            The optional os.features field specifies an array of strings,
+            each listing a required CPU feature (for example on Windows
```

Comment:
I'm a little confused here. Are these CPU features or features provided by the os? Maybe some documentation on this would help to clarify.